waspc/src/Wasp/Generator/WebAppGenerator/NpmInstall.hs

```diff
@@ -9,7 +9,7 @@ import qualified Wasp.Generator.Job as J
 import Wasp.Generator.Job.Process (runNodeCommandAsJob)
 import qualified Wasp.Generator.WebAppGenerator.Common as Common
 
-setupWebApp :: Path' Abs (Dir ProjectRootDir) -> J.Job
-setupWebApp projectDir = do
+npmInstallClient :: Path' Abs (Dir ProjectRootDir) -> J.Job
+npmInstallClient projectDir = do
```

Comment:
Same as above, I would name this `installNpmDeps`.

Regarding the rename from `webapp` to `client` -> I am in for it, although I wouldn't do it only partially like this, because now we are mixing WebApp and Client and that makes it even more confusing in my opinion. Instead I would rename it all at once.

We might also want to consider naming it `webClient`, not just `client`, in case in the future we add more types of clients. But that might be overengineering, true.